### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by Leon-Arno**
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `master` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `master` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `master`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](https://classroom.github.com/help/leaving-feedback-in-github)”.
</details>

Subscribed: @MarianaPantelic

### CODE REVIEW

**Comment by marko-dci** [comment]
1. [The class float](https://github.com/FbW43-2/weekend-exercises-MarianaPantelic/blob/a85b5c580d6aa4f52284cc3195f663c3cdc54449/index.html#L24) could be renamed into something more explicative
2. There is no [class header in your html](https://github.com/FbW43-2/weekend-exercises-MarianaPantelic/blob/a85b5c580d6aa4f52284cc3195f663c3cdc54449/css/style.css#L19)
3. the [border properties ](https://github.com/FbW43-2/weekend-exercises-MarianaPantelic/blob/a85b5c580d6aa4f52284cc3195f663c3cdc54449/index.html#L10)could be applied directly to the header element for now

All good for the rest

**Comment by MarianaPantelic** [comment]
Made the requested changes

File: css/style.css [hunk 1]
```diff
@@ -27,7 +27,7 @@ footer {
   color: white;
   padding
 }
-.float {
+.right-align {
```
Comment (marko-dci) [inline css/style.css]: Sorry, what I meant was: you should make clear that this class will move the element on the right side..

e.g: 'float-right', 'align-right', ecc... something like that

**Review by marko-dci — CHANGES_REQUESTED** [review]

File: css/style.css (hunk 1 above)
Comment (MarianaPantelic) [reply css/style.css]: so i changed it to .right-align, the same thing as .align-right

**Comment by MarianaPantelic** [comment]
Made the requested change

**Pull request merged**